sycl/doc/extensions/supported/sycl_ext_oneapi_backend_level_zero.md

```diff
@@ -388,6 +411,47 @@ make_buffer(
 Construct a SYCL buffer instance from a pointer to a Level Zero memory allocation. Please refer to <code>make_buffer</code>
 description above for semantics and restrictions.
 The additional <code>AvailableEvent</code> argument must be a valid SYCL event. The instance of the SYCL buffer class template being constructed must wait for the SYCL event parameter to signal that the memory native handle is ready to be used.
+</tr>
+
+<tr>
+<td>
+
+``` C++
+make_image(
+    const backend_input_t<backend::ext_oneapi_level_zero,
+                          image<Dimensions, AllocatorT>> &,
+    const context &Context, event AvailableEvent = {})

```

Comment:
I don't think we should have a default `event` parameter like this.  It seems better to follow `make_buffer` and have two overloads.

Reply:
Why is that better?   
The `make_buffer` implementation is not so simple. It uses an optional event for `AvailableEvent` in the generic backend declaration, and only the LevelZero specialization has two overloads, and then second overload just calls a helper function with a default event in the "eventless" variant.  
The `make_image` implementation doesn't need a helper, and for that it seems more consistent and simpler to have the optional `AvailableEvent` in all declarations and specializations.